setup.py

```diff
@@ -5,14 +5,16 @@
     include_package_data=True,
     long_description_content_type="text/markdown",
     install_requires=[
-        "dbus-python==1.2.16",
```

Comment:
Can you explain your rational on removing this dependency? I believe this is required for `bluez.py` to function since we need a binding for BlueZ-based dbus communication.

Reply:
> Can you explain your rational on removing this dependency? I believe this is required for `bluez.py` to function since we need a binding for BlueZ-based dbus communication.

A lot of people were reporting that during install, linux couldn't build a wheel for dbus-python, but simply removing it from the dependancy list still works

Reply:
Unfortunately, I'm fairly certain that this dependency can't be dropped. I don't believe current installations of Python on most Linux environments provide ways to interface with the dbus otherwise. I've performed a fresh install with your PR branch and I'm getting a `ModuleNotFoundError` for the dbus import.

Steps to Reproduce:
```bash
git clone https://github.com/Dummydud3/nxbt
cd nxbt
git checkout pull
sudo -i # For easier root operation and virtual environment use/creation
python3 -m venv venv # Assuming a full python install with venv
source venv/bin/activate
pip install -e .
nxbt test
> ModuleNotFoundError: No module named 'dbus'
```

I'm using a virtual environment above to ensure we're working with a clean slate on install (e.g. no pre-installed packages from previous pip usage in other projects).

My hunch is that a pre-installed copy of dbus-python (from a previous NXBT install, maybe?) is tagging along in instances where excluding this dependency works. The issues where users are having trouble might have to do with the lack of proper dbus bindings (e.g. pkg-config build-essential libdbus-glib-1-dev libgirepository1.0-dev) or missing Python headers. Most Linux distributions will come with these packages but some more minimal distros might not. This is probably something that can get clarified in the README further and possibly a check can get added to the project itself.